QModManager/API/ModLoading/Internal/PatchingOrder.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace QModManager.API.ModLoading.Internal
+{
+    /// <summary>
+    /// FOR INTERNAL USE ONLY.
+    /// </summary>
+    public enum PatchingOrder
```

Comment:
This one is `public` because it's being used by the [IPatchOrder](https://github.com/QModManager/QModManager/blob/dev/ModLoading/QModManager/API/ModLoading/Internal/IPatchMethod.cs) interface.  
And interface method implementations need to be public.  
I know it's not ideal to have this exposed, but there really isn't anything an external assembly could do with this.  
The various [QModdPatch](https://github.com/QModManager/QModManager/blob/dev/ModLoading/QModManager/API/ModLoading/QModPatchMethod.cs) methods don't even allow this to be changed.

Reply:
I do not think that there is a need for the `IPatchMethod` interface.

Reply:
I see a way to do this without that interface.  
Yes, this public enum can be made internal with just a little rework of the patch method attributes.